<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtSqlQueryWidget</class>
 <widget class="QWidget" name="mdtSqlQueryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbables">
       <property name="text">
        <string>Tables</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pbTables">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbFields">
       <property name="text">
        <string>Fields</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pbFields">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbSort">
       <property name="text">
        <string>Sorting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pbSort">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbFilters">
       <property name="text">
        <string>Filters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pbFilters">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="twData"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
